### DISCUSSION TITLE
Power on at 12:00 each day. Take 1 photo. Power off again. Loop endlessly.

### DISCUSSION THREAD

**Discussion by Soundblaster1** [gopro/labs#996] [Q&A]
Hi,
Been looking at Labs for a while, and also the visual script editor, but without success.
What i wanted to achieve is to get the Hero11Black to boot at 12:00, take one single photo, then shut down, and repeat each day until i stop it manually.
Tried this: dP!12:00S!1N!R but still does not seem to work.

Any good tips?
Thanks :)

**Comment by dnewman-gpsw** [comment]
What happens? Share the log file from MISC/qrlog.txt

The script looks mostly fine and should work.  The ```dP``` doesn't need to be in the loop.

This also should work: ```!12:00NmP!1S!1N!R```
